controlplane/internal/controllers/rke2controlplane_controller.go

```diff
@@ -483,35 +500,17 @@ func (r *RKE2ControlPlaneReconciler) reconcileDelete(ctx context.Context, cluste
 	return ctrl.Result{RequeueAfter: deleteRequeueAfter}, nil
 }
 
-// 	// TODO: Improve this part once there are dependencies on the Control Plane Object!
-// 	var ok bool = true
-// 	for _, finalizer := range rcp.GetObjectMeta().GetFinalizers() {
-// 		if ok {
-// 			ok = controllerutil.RemoveFinalizer(rcp, finalizer)
-// 		}
-// 	}
-// 	if !ok {
-// 		logger.Info("unable to remove all finalizers")
-// 		err = fmt.Errorf("unable to remove all finalizers")
-// 		res = ctrl.Result{}
-// 		return
-// 	}
-
-// 	if err = r.Delete(ctx, rcp); err != nil {
-// 		res = ctrl.Result{RequeueAfter: 2 * time.Minute}
-// 		return
-// 	}
-
-// 	return ctrl.Result{}, nil
-// }
+// 	TODO: Improve this part once there are dependencies on the Control Plane Object!
```

Comment:
Need an issue for this?

Reply:
I will create an issue to see if this is still needed.